### PR TITLE
Deck descriptions

### PR DESCRIPTION
**Pull request by alexmiddeleer**
Allows users to add a deck description on the `/deck/:id` page

### CODE REVIEW

File: next/pages/deck.js [hunk 1]
```diff
@@ -46,7 +46,9 @@ export default withRouter(({ router }) => {
 
   return (
     <Layout title={title} desc={description}>
-      <PageBanner image={PageBanner.IMG_DECKS} url="/decks">Decks</PageBanner>
+      <PageBanner image={PageBanner.IMG_DECKS} url="/decks">
```
Comment (lily-pytel) [inline next/pages/deck.js]: LOL

File: next/components/deck-description-edit.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+          placeholder={placeholder}
+          onChange={onChangeDeckDesc}
+        ></textarea>
+      </div>
+    </div>
+  );
+
+  const editButton = (
+    <div>
+      <style jsx>{`
```
Comment (lily-pytel) [inline next/components/deck-description-edit.js]: I would personally add a bit of margin top here for some space

**Comment by lily-pytel** [comment]
I don't know why the tests are failing because of stuff in another branch, but I would pull from the publish mode branch and make it the parent of this. I fixed some props warnings but it required some tests updates and stuff... somehow looks like Travis is taking it out on your branch

**Comment by alexmiddeleer** [comment]
Yep no worries, I saw it failed after I rebased.

**Pull request merged**